### ISSUE TITLE
[a11y] Leadership page accessibility improvements

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15335]
# Axe-core® Accessibility Results for /en-US/about/leadership/

Page URL: https://www-dev.allizom.org/en-US/about/leadership/

## axe-core found 4 violations

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Elements must only use supported ARIA attributes | aria-allowed-attr | WCAG 2 Level A, WCAG 4.1.2 | critical | 1   |
| [2](#2) | ARIA attributes must conform to valid names | aria-valid-attr | WCAG 2 Level A, WCAG 4.1.2 | critical | 1   |
| [3](#3) | Heading levels should only increase by one | heading-order | Best practice | moderate | 1   |
| [4](#4) | Page should contain a level-one heading | page-has-heading-one | Best practice | moderate | 1   |

### Failed

1. Elements must only use supported ARIA attributes

[Learn more](https://dequeuniversity.com/rules/axe/4.10/aria-allowed-attr?application=playwright)

aria-allowed-attr

WCAG 2 Level A, WCAG 4.1.2

Ensure an element's role supports its ARIA attributes

critical

Issue Tags: cat.aria wcag2a wcag412 EN-301-549 EN-9.4.1.2

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-sidemenu-main<br>```<br><br>**Element source**<br><br>```<br><section class="mzp-c-sidemenu-main" aria-expanded="false"><br>``` | Fix all of the following:<br><br>*   ARIA attribute is not allowed: aria-expanded="false" |

2. ARIA attributes must conform to valid names

[Learn more](https://dequeuniversity.com/rules/axe/4.10/aria-valid-attr?application=playwright)

aria-valid-attr

WCAG 2 Level A, WCAG 4.1.2

Ensure attributes that begin with aria- are valid ARIA attributes

critical

Issue Tags: cat.aria wcag2a wcag412 EN-301-549 EN-9.4.1.2

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>#laura-chambers<br>```<br><br>**Element source**<br><br>```<br><article id="laura-chambers" class="vcard has-bio" itemscope="" itemtype="http://schema.org/Person" aria-role="button" tabindex="0"><br>``` | Fix any of the following:<br><br>*   Invalid ARIA attribute name: aria-role |

3. Heading levels should only increase by one

[Learn more](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright)

heading-order

Best practice

Ensure the order of headings is semantically correct

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-sidemenu-title<br>```<br><br>**Element source**<br><br>```<br><h4 class="mzp-c-sidemenu-title "><br>          <a href="https://www-dev.allizom.org/en-US/about/leadership/">Mozilla Leadership</a><br>        </h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |

4. Page should contain a level-one heading

[Learn more](https://dequeuniversity.com/rules/axe/4.10/page-has-heading-one?application=playwright)

page-has-heading-one

Best practice

Ensure that the page, or at least one of its frames contains a level-one heading

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>html<br>```<br><br>**Element source**<br><br>```<br><html class="windows js windows-10-plus x64 is-modern-browser mzp-has-sticky-navigation loaded" lang="en" dir="ltr" data-country-code="US" data-needs-consent="False" data-latest-firefox="131.0.3" data-esr-versions="128.3.1" data-gtm-container-id="GTM-MW3R8V" data-stub-attribution-rate="1.0" data-sentry-dsn="https://c1f94ae1b8c54ed0adcf223328157caf@o1069899.ingest.sentry.io/6260211"><br>``` | Fix all of the following:<br><br>*   Page must have a level-one heading |
